doc/developer/debugging.rst

```diff
@@ -53,10 +57,8 @@ For better security, the session cookie is accessible only via http protocol (me
 and is secure (meaning, the cookie is transmitted only in https requests, not in http requests).
 For this reason, you should have your application running on https also in your development environment.
 
-To achieve that, if it is not already done, copy the file ``docker-compose.override.sample.yaml``
-to ``docker-compose.override.yaml``.
-
-Then access the application on `https://localhost:8484/ <https://localhost:8484/>`_.
+With the default configuration in the file ``docker-compose.override.yaml`` your application will be
+abailable in ``https``, and the authentication will work.
```

Comment:
abailable -> available

and the authentication will work. -> and authentication is available.

Reply:
For me the work is better because before the authentication is available but is not working...

Reply:
But then it is more like a bug.
If the authentication is available but can not work in http, then it is useless (in http)

Reply:
No it's not an issue that's an application made to work in https don't work on http...

Reply:
But I think we do not explain anywhere this general design decision of GMF - the decision that authentication is only available in https (or that authentication will only work in https, whatever you prefer).
And this is quite new, only since 2.4 if I understand correctly. I guess it should be explained in https://camptocamp.github.io/c2cgeoportal/master/integrator/security.html.